src/services/communications/Ethernet_packets.py

```diff
@@ -0,0 +1,52 @@
+import struct
+
+
+class Packets:
+    def __init__(self,*packets): 
+        #packets is a list where first goes an ID uint16_t and then string with types
+        self._dict_packets = {}
+        for packet in packets:
+            if len(packet) < 1:
+                raise ValueError ("Every packet need a ID at least")
+            if not isinstance(packet[0],int) or not (0 <= packet[0] <= 0xFFFF):
+                raise ValueError(f"The id must be an uint16_t, {packet[0]}")
```

Comment:
make a function to check this but for every type, that way you can also use it to validate that the values you are using to generate the packets are in range